tutorials/buoyantSimpleFoam-aster/precice-config.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<precice-configuration>
+  <log>
+    <sink enabled="true" output="stdout" type="stream" filter= "(%Severity% >= trace)"/>
```

Comment:
Please use this updated format: https://github.com/precice/tutorials/blob/develop/FSI/flap_perp/OpenFOAM-CalculiX/precice-config_serial.xml#L6

Reply:
Format is updated.